.github/workflows/ci.yml

```diff
@@ -345,3 +345,70 @@ jobs:
               --disable-public-ips \
               --additional-experiments=use_prime \
               --parameters=batch-size="${BATCH_SIZE}",events-project-id="${ENV_PROJECT_ID}",events-table="${EVENTS_TABLE}",leech-project-id="${ENV_PROJECT_ID}",leech-table="${ARTIFACTS_TABLE}",logs-bucket-name="${LOGS_BUCKET_NAME}",github-app-id="${GH_APP_ID}",github-install-id="${GH_INSTALL_ID}",github-private-key-secret="${GH_PK_SECRET}"
+
+  deployment_commit_review_status_pipeline_autopush:
```

Comment:
It's probably getting to the point where we should parameterize these deployments into a reusable action or a matrix deploy. This can be a follow up.

Reply:
Sound good.